cpp/diffraction/StaticStructureFactorDebye.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2010-2020 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#ifndef STATIC_STRUCTURE_FACTOR_DEBYE_H
+#define STATIC_STRUCTURE_FACTOR_DEBYE_H
+
+#include "Box.h"
+#include "Histogram.h"
+#include "NeighborQuery.h"
+
+/*! \file StaticStructureFactorDebye.h
+    \brief Routines for computing static structure factors.
+
+    Computes structure factors from the Fourier transform of a radial distribution function (RDF).
+    This method is not capable of resolving k-vectors smaller than the magnitude 4 * pi / L, where L is the
+    smallest side length of the system's periodic box.
+*/
+
+namespace freud { namespace diffraction {
+
+class StaticStructureFactorDebye
+{
+    using StaticStructureFactorDebyeHistogram = util::Histogram<float>;
```

Comment:
Doesn't this using statement just make everything unnecessarily verbose? I would suggest shortening the name somewhat.

Reply:
I've changed the name to S_kHistogram.